demo/utils/definition/table-definition/table-definition.js

```diff
@@ -4,6 +4,19 @@ import OptionsHelper from '../../../../src/utils/helpers/options-helper';
 export default (definition) => {
   definition.type = 'grids';
   definition.propValues = assign({}, definition.propValues, {
+    actions: "[{ icon: 'bin' }, { icon: 'settings' }]",
+    actionToolbarChildren: `(context) => {
```

Comment:
Do you want to show an example where you are getting the selected value out of context

Reply:
there is one on line 10 and line 13?

Reply:
I meant he object of selected values rather than just the disabled flag